packages/core/src/components/dialog/dialog.tsx

```diff
@@ -146,7 +146,7 @@ export class Dialog extends AbstractPureComponent<DialogProps> {
                     <div
                         className={classNames(Classes.DIALOG, className)}
                         role={role}
-                        aria-modal={overlayProps.enforceFocus ?? Overlay2.defaultProps?.enforceFocus}
```

Comment:
Would it be useful to have `Overlay2` still export its default props if we need to reference them internally?

```tsx
// Overlay2.tsx

export const OVERLAY2_DEFAULT_PROPS = {
  enforceFocus: true
};
```

I think that makes it easier to see the intent of the code here. Otherwise it's more difficult to know why someone chose to default `aria-modal` to `true`.

Reply:
Yep, can do! I 👍 had considered something like this originally, but wavered at the time of implementing. I think this is a good idea for posterity since these can technically be consumed externally. Having the export set of values to point to will give us a migration path in case there are any other consumers we are unaware of.

Reply:
5084a87cee82f326fafdff3f6e0f7f414b221364